### DISCUSSION TITLE
Navigation from other location than current

### DISCUSSION THREAD

**Discussion by mrkbrnd-droid** [osmandapp/OsmAnd#16863] [Maps Related]
Hi,

I used to be able to generate a route trough the navigation function by choosing a from- and to-location other than my current location. I am currently in a train using OsmAnd for Android. Whenever I select a random location as starting point other than my current position, it always replaces it with my current position when starting to calculate.

I'm sure I was able to do this before. Am I doing something wrong? Tried by putting phone on airplane-mode and then calculation of the route kept hanging.

Mark

**Comment by pebogufi** [comment]
As I do it:
FIRST SWITCH OFF GPS !
Then select destination
Then select start point
DON'T press START button.

If gps is active, your current pos will be used as start point after selecting destination, route will be calculated immediately. 
Of course you can change start point, then the route will be new calculated with this new start point.

**Comment by tommycrock** [comment]
Or use plan a route?

**Comment by mrkbrnd-droid** [comment]
Thanks. Weird thing is, when I tried recently, OsmAnd seemed not to immediately replace the from-location with my current location when starting navigation while I was in a moving vehicle. Will keep an eye on it.
